datamade/la-metro-councilmatic
# Containerize application for local development

## Overview

This PR:

- Adds a `docker-compose.yml` file for use in local development.
- Moves the existing `docker-compose.yml` contents to a `docker-compose.deployment.yml` file, for easy differentiation.
- Updates the deployment scripts to `docker-compose up` from the deployment file.

### Checklist


### Demo

N/A

## Testing Instructions

- View the updated README.md and confirm that markup looks good and the instructions make sense.
- Follow the instructions as outlined in the revised Setup section of the README and confirm that you are able to launch the app! **Run `import_data` with `--update_since=2019-08-01` for quicker startup time.**
